feature/security/gnsi/credentialz/tests/hiba_authentication/README.md

```diff
@@ -0,0 +1,60 @@
+# Credentialz-5: Hiba Authentication
+
+## Summary
+
+Test that Credentialz properly configures (hiba) certificate authentication.
+
+
+## Procedure
+
+* Prior to writing test the following steps were taken to create appropriate keys/certs.
+  * Build hiba
```

Comment:
I guess this is also partly outdated -- for context: originally I had just generated all the keys/certs in a container and then just dragged them over to my fp stuff. the thought here was this would be easier if folks haven't built hiba, they could still run the test. in a later commit I made the test generate keys/certs if hiba is on path, then removed the stuff I had generated (with like super crazy long lifetimes :)) from the repo.

so, this part can probably just go away anyway and/or this file can depending on the other thread.

Reply:
@sourcequench, would it be preferable to simply have something like:
"Prior to writing test, the [following steps (HIBA Docs)](https://github.com/google/hiba/blob/main/CA.md) are to be taken to create appropriate keys/certs.

Reply:
I have a plain vanilla docker image publicly available for this very use case - it does everything it can to set up the CA in exactly the steps in the hiba doc and is ready to go.

docker pull \
    us-west1-docker.pkg.dev/authentication-images/hiba/hiba-ca:latest

Reply:
Would Google like to revise/update the original README and update accordingly?

Reply:
Meaning this bit? feature/security/gnsi/credentialz/tests/hiba_authentication/README.md

"Follow the instructions for setting up a HIBA CA" is really what we intend. I wasn't saying that we need to use that docker container, just offering it as a standardized way to test. That container is exactly that, a built hiba CA that follows those instructions.

Reply:
Updated this line to reflect the original Readme [statement](https://github.com/openconfig/featureprofiles/tree/main/feature/security/gnsi/credentialz/tests#setup-4)